Modules/Core/Transform/include/itkScaleSkewVersor3DTransform.h

```diff
@@ -121,11 +121,11 @@ class ITK_TEMPLATE_EXPORT ScaleSkewVersor3DTransform : public VersorRigid3DTrans
 
   /** Set the transformation from a container of parameters
    * This is typically used by optimizers.
-   * There are 15 parameters:
+   * There are 12 parameters:
    *   0-2   versor
```

Comment:
Isn't versor a 4-value thing?

Reply:
P.S.: `right part of the versor` is written in the implementation file

Reply:
I've added text to clarify this.  Thanks!